webapp/app.py

```diff
@@ -64,6 +65,14 @@
 discourse_docs.init_app(app)
 
 
+@app.route("/docs/_cache_clear")
+def cache_clear():
+    cached_urls = len(list(docs_session.cache.urls))
+    docs_session.cache.clear()
+    hostname = socket.gethostname()
+    return f"{hostname}: OK, all {cached_urls} URL(s) ☢🚀 from the cache", 410
```

Comment:
I am not sure a 410 status code is the best one:

> A 410 response is cacheable by default.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/410

Reply:
What do you suggest? 200 GET is also cacheable by default...

Reply:
Well, I also mean why a 410? a 410 is similar to a 404 but more intentional. a `200` makes more sense with the right cache headers for this case than a 410.